apps/mobile-wallet/package.json

```diff
@@ -48,7 +48,7 @@
     "big-integer": "^1.6.51",
```

Comment:
(Unrelated to this line, just making a thread)

1. The width of the input changes while typing. Is it intended?
2. Button to continue is missing

<img width="350" alt="image" src="https://github.com/user-attachments/assets/13575ab7-cbaa-4f7b-957c-55057486367b">

Reply:
@mvaivre is this intended or should it be fixed? I can work on it.

Reply:
I'll fix it